tests/cupy_tests/manipulation_tests/test_add_remove.py

```diff
@@ -278,33 +279,35 @@ def test_unique_equal_nan(self, xp, dtype, equal_nan):
         return xp.unique(a, axis=0, equal_nan=equal_nan)
 
     @testing.with_requires("numpy>=2.0")
-    @pytest.mark.parametrize("attr", ["values", "indices", "inverse_indices", "counts"])
+    @pytest.mark.parametrize(
+        "attr", ["values", "indices", "inverse_indices", "counts"]
+    )
     @testing.for_all_dtypes(no_float16=True, no_bool=True, no_complex=True)
     @testing.numpy_cupy_array_equal()
-    def test_unique_all(self, xp, dtype, attr):
+    def test_v2api_unique_all(self, xp, dtype, attr):
         a = testing.shaped_random((100, 100), xp, dtype)
         return getattr(xp.unique_all(a), attr)
 
     @testing.with_requires("numpy>=2.0")
     @pytest.mark.parametrize("attr", ["values", "counts"])
     @testing.for_all_dtypes(no_float16=True, no_bool=True, no_complex=True)
     @testing.numpy_cupy_array_equal()
-    def test_unique_counts(self, xp, dtype, attr):
+    def test_v2api_unique_counts(self, xp, dtype, attr):
```

Comment:
Could you restore the name of the test to `test_unique_counts` and rename the tests for the legacy API to `test_return_unique_counts`?

Reply:
How about `test_unique_return_counts` ?

I feel this naming is more consistent with other tests/argument names.

Reply:
Sorry, it's a typo. `test_unique_return_counts` is preferred.

Reply:
Fixed in 249bb1f09af708ad2abb9033d9ee79b7712c474e .